backend/src/main/java/com/zeros/backend/AppConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.zeros.backend;
+
```

Comment:
Simple question : what does zeros stand for ?

Reply:
@NaitYoussef  i start with a sample a project and forget it :(
can you suggest another name ( we will refactor it )